.github/workflows/linters.yml

```diff
@@ -20,6 +20,20 @@ jobs:
           [ -f .rubocop.yml ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/ror/.rubocop.yml
       - name: Rubocop Report
         run: rubocop --color
```

Comment:
<kbd>OPTIONAL</kbd> maybe you can make sure that the linters.yml file is set up correctly with the correct code, or delete and set up the linters from scratch to avoid the error.